src/popup/router/pages/CommentNew.vue

```diff
@@ -42,14 +41,15 @@ export default {
   methods: {
     async sendComment() {
       this.loading = true;
+      const postData = [
+        this.id,
+        this.text,
+        await this.sdk.address(),
+        async data => Buffer.from(await this.sdk.signMessage(data)).toString('hex'),
+        this.parentId,
+      ];
       try {
-        await Backend.sendTipComment(
-          this.id,
-          this.text,
-          await this.sdk.address(),
-          async data => Buffer.from(await this.sdk.signMessage(data)).toString('hex'),
-          this.parentId,
-        );
+        this.$store.dispatch('sendTipComment', postData);
```

Comment:
```suggestion
        await this.$store.dispatch('sendTipComment', [
         this.id,
         this.text,
         await this.sdk.address(),
         async data => Buffer.from(await this.sdk.signMessage(data)).toString('hex'),
         this.parentId,
       ]);
```
- await is missing
- don't think that is needed to have intermediate variable (also `postData` name is so unmeaningful in this context)